packages/paste-website/src/components/DoDont.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Text} from '@twilio-paste/text';
+
+const StyledWrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  grid-column-gap: 24px;
+`;
+
+interface DoDontProps {
+  children?: React.ReactNode;
```

Comment:
The `?` here isn't necessary.  `React.ReactNode` includes undefined as an option.